lms/djangoapps/grades/api/urls.py

```diff
@@ -0,0 +1,11 @@
+""" Grades API URLs. """
+from django.conf.urls import (
+    include,
+    patterns,
+    url,
+)
+
+urlpatterns = patterns(
+    '',
+    url(r'^v0/', include('lms.djangoapps.grades.api.v0.urls', namespace='v0')),
```

Comment:
Nit: I'm not much in favor of putting the version number in the directory structure.  There's no need to assume that a new folder is needed for each new version of the API.  If it's not too much work, can you move the files in the v0 directory to the parent grades/api directory?